### ISSUE TITLE
Improper cursor placement on multi-hierarchical bullet lists

### ISSUE THREAD

**Issue by transcendr** [superlistapp/super_editor#872]
When you add a second level of bullets, the cursor starts at the beginning of the line instead of after the bullet.  Only after you begin typing again will the cursor snap back into place.

https://user-images.githubusercontent.com/37011672/204637114-5601d515-2247-44ef-a1b0-3fc6bde67c73.mp4

**Comment by matthew-carroll** [comment]
I tried to reproduce this in the example app on Mac and web but didn't notice any issue with caret placement.

@transcendr can you confirm whether this is still a problem on `main`?

**Comment by matthew-carroll** [comment]
This seems to be a duplicate of https://github.com/superlistapp/super_editor/issues/904 - closing in favor of that one.
